trade_management_unit/migrations/0003_create_termination_algorithm.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2 - Create Termination Algorithm Table
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('trade_management_unit', '0002_create_initiation_algorithm'),
+    ]
+
+    operations = [
+        # Create TerminationAlgorithm table
+        migrations.CreateModel(
+            name='TerminationAlgorithm',
+            fields=[
+                ('id', models.AutoField(primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=255, unique=True, db_index=True)),
+                ('display_name', models.CharField(max_length=255, db_index=True, null=True, blank=True)),
+                ('description', models.TextField()),
+                ('is_active', models.BooleanField(default=True, db_index=True)),
+                ('parameters_schema', models.JSONField(default=dict, blank=True, help_text='Algorithm configuration parameters')),
+                ('created_at', models.DateTimeField(default=django.utils.timezone.now)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+            ],
+            options={
+                'verbose_name': 'Termination Algorithm',
+                'verbose_name_plural': 'Termination Algorithms',
+                'db_table': 'termination_algorithms',
+                'ordering': ['name'],
+            },
+        ),
+
+        # Add performance indexes for termination_algorithms table
+        migrations.RunSQL(
+            """
+            -- Indexes for termination_algorithms
+            CREATE INDEX idx_termination_algorithms_name ON termination_algorithms(name);
+            CREATE INDEX idx_termination_algorithms_is_active ON termination_algorithms(is_active);
+            CREATE INDEX idx_termination_algorithms_created_at ON termination_algorithms(created_at);
+            CREATE INDEX idx_termination_algorithms_display_name ON termination_algorithms(display_name);
```

Comment:
trade_management_unit/migrations/0003_create_termination_algorithm.py
 Let's not have index over the display name because we will hardly query over the display name here